go.mod

```diff
@@ -25,8 +25,91 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
```

Comment:
Why did a second require section get added to the go.mod?

Reply:
go 1.17 does this.  First I'd seen it, too.